### PR TITLE
R4R: Fix the Getting-Started section of docs

### PR DESCRIPTION
**Pull request by shilei-wang**
Fix the getting-started section of docs documentation

### CODE REVIEW

File: docs/get-started/Full-Node.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Setup A Full-node
```
Comment (haifengxi) [inline docs/get-started/Full-Node.md]: Setup A Full Node

Comment (shilei-wang) [reply docs/get-started/Full-Node.md]: fixed

File: docs/zh/get-started/Validator-Node.md [hunk 2]
```diff
@@ -97,25 +99,17 @@ iriscli stake validator  {address-validator-operator}  --chain-id={chain-id} --n
...
-iriscli stake edit-validator --from= {val-name}  --moniker="choose a moniker"  --website="https://irisnet.org"  --details="team" --chain-id=irishub-8000 
-   --node=tcp://localhost:26657 --fee=0.3iris  
-```
-### 查询验证人信息
-
-你可以通过以下命令查询验证人的信息：
-
-```
-iriscli stake validator {address-validator-operator} --chain-id=<chain-id> 
+iriscli stake edit-validator --from=<key name> --moniker=<your_custom_name> --website=<your_website> --details=<your_details> --chain-id=<chain-id> --node=tcp://localhost:26657 --fee=0.3iris
```
Comment (chengwenxi) [inline docs/zh/get-started/Validator-Node.md]: Maybe we should add some descriptions about `identity` when creating and editing validators.

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -167,11 +167,10 @@ module.exports = {
...
                     '/zh/get-started/': [
-                        ['Download-Rainbow.md', 'Download Rainbow'],
-                        ['Install-the-Software.md', 'Install the Software'],
                         ['Join-the-Testnet.md', 'Use the Testnet'],
-                        ['Join-the-Mainnet.md', 'Join the Mainnet']
-
+                        ['Join-the-Mainnet.md', 'Join the Mainnet'],
+                        ['Full-Node.md', 'Full Node'],
+                        ['Validator-Node.md', 'Validator Node']
                     ],
```
Comment (haifengxi) [inline docs/.vuepress/config.js]: The left-side menu items should be translated into Chinese here, e.g.,
Use the Testnet --> 使用测试网

File: docs/get-started/README.md [hunk 4]
```diff
@@ -2,7 +2,13 @@
 
 ## What is IRISnet
 
-IRIS network (a.k.a. IRISnet) is named after Greek goddess **Iris**, said to be the personification of the rainbow and the faithful messenger between heaven and humanity. IRIS network aims to build the foundation which facilitates construction of distributed business applications. IRIS hub will provide iServices, which allow resources such as data service and computing services being invoked across blockchains. To know more about IRISnet, please read this blog.
+IRIS network (a.k.a. IRISnet) is named after Greek goddess **Iris**, said to be the personification of the rainbow and the faithful messenger between heaven and humanity. IRIS network aims to build the foundation which facilitates construction of distributed business applications. IRIS hub will provide iService, which allow resources such as data service and computing services being invoked across blockchains. To know more about IRISnet, please read this blog.
+
```
Comment (haifengxi) [inline docs/get-started/README.md]: IRIS Network (a.k.a IRISnet) is named after...

IRISnet aims to provide a foundation **that** facilitates ...

IRIS Hub (a.k.a IRIShub), which is the centerpiece of IRISnet, introduces a unified service model that allows resources such as data **services** and computing services **to be** invoked ...


URL to the blog is missing.

File: docs/get-started/README.md [hunk 5]
```diff
@@ -2,7 +2,13 @@
...
+## How to install IRIShub  
+
```
Comment (haifengxi) [inline docs/get-started/README.md]: How to install `iris`

File: docs/get-started/README.md [hunk 6]
```diff
@@ -2,7 +2,13 @@
...
+
+Please follow this [instruction](../software/How-to-install-Irishub.md) to install IRIShub.
+
```
Comment (haifengxi) [inline docs/get-started/README.md]: to install `iris`

File: docs/get-started/README.md [hunk 7]
```diff
@@ -2,7 +2,13 @@
...
 
 ## How to Join IRIShub  Mainnet
```
Comment (haifengxi) [inline docs/get-started/README.md]: How to Join the Mainnet

File: docs/get-started/Validator-Node.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Running a Validator Node
 
-Before setting up your validator node, make sure you've already installed  **Iris** by this [guide](Full-Node.md)
+Before setting up your validator node, make sure you've already installed  `Iris` by this [guide](Full-Node.md) and your node is fully synced.
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: already installed `iris` by **following** this guide...

File: docs/get-started/Validator-Node.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
 
 Validators are responsible for committing new blocks to the blockchain through consensus. A validator's stake will be slashed if they become unavailable, double sign a transaction, or don't cast their votes. Please read about Sentry Node Architecture to protect your node from DDOS attacks and to ensure high-availability.
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: ...will be slashed if **it becomes** unavailable, **double-signs** a transaction, or **doesn't** cast votes...

File: docs/get-started/Validator-Node.md [hunk 10]
```diff
@@ -26,21 +26,21 @@ It is the only way to recover your account if you ever forget your password.
...
 
 The seed phrase of this account will also be displayed. You could use these 24 phrases to recover this account in another server. The recover command is:
 ```
-iriscli keys add <NAME_OF_KEY> --recover
+iriscli keys add <key_name> --recover
 ```
 
 
-### Claim tokens
+### Claim tokens (Only for fuxi Testnet)
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: please change "fuxi Testnet" to "Fuxi Testnet" throughout the doc.

File: docs/get-started/Validator-Node.md [hunk 11]
```diff
@@ -26,21 +26,21 @@ It is the only way to recover your account if you ever forget your password.
...
 
-You can always get some `IRIS`  by using the [Faucet](https://testnet.irisplorer.io/#/faucet). The faucet will send you 10IRIS every request, Please don't abuse it.
+You can always get some `IRIS`  by using the [Faucet](https://testnet.irisplorer.io/#/faucet). The faucet will send you 10IRIS for every request, please don't abuse it.
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: You can always get some test tokens by using ...

File: docs/get-started/Validator-Node.md [hunk 12]
```diff
@@ -26,21 +26,21 @@ It is the only way to recover your account if you ever forget your password.
...
 
-Once you have created your own address, please  then you could use this　account to stake as a validatord. The following command is used to check the balance of your account:
+Once you have created your own address,  then you could use this account to stake as a validator. The following command is used to check the balance of your account:
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: Once you ..., you can use it to ...

File: docs/get-started/Validator-Node.md [hunk 13]
```diff
@@ -55,48 +55,42 @@ iriscli status --node=tcp://localhost:26657
 
 You should also be able to see `catching_up` is `false`. 
 
-You need to get the public key of your node before upgrade your node to a validator node. The public key of your node starts with `icp`, 
-it can be used to create a new validator by staking tokens. To understand more about the address encoding in IRISHub, 
+You need to get the public key of your node before upgrade your node to a validator node. The public key of your node starts with `icp`, it can be used to create a new validator by staking tokens. To understand more about the address encoding in IRISHub,
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: ...before **upgrading** your node...

IRISHub --> IRIShub

File: docs/get-started/Validator-Node.md [hunk 14]
```diff
@@ -55,48 +55,42 @@ iriscli status --node=tcp://localhost:26657
...
 
 ```
-iriscli stake create-validator --chain-id=<chain-id> --from=<key name> --fee=0.3iris --pubkey=<pubkey> --amount=1iris --moniker={validator-name} --commission-rate=0.1
+iriscli stake create-validator --chain-id=<chain-id> --from=<key name> --gas=100000 --fee=0.6iris --pubkey=<validator public key> --amount=10iris --moniker=<your_custom_name> --commission-rate=0.1
 ```
-Don't forget the `fee` and `gas` field.  To read more about coin-type in IRISHub, you should read [this](../features/basic-concepts/coin-type.md)
-
+Please note the `fee` and `amount` can be the **decimal** of IRIS token, like `1.01iris`. And you could also use other coin-type like `iris-milli`, To read more about coin-type in IRISHub, you should read [this](../features/basic-concepts/coin-type.md)
 
+To read more about fee mechanism in IRISHub, go to this [doc](../features/basic-concepts/fee.md)
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: IRISHub --> IRIShub throughout the doc

File: docs/get-started/Validator-Node.md [hunk 15]
```diff
@@ -117,17 +111,10 @@ You can edit your validator's public description following [this](../cli-client/
...
-### View Validator Description
+iriscli stake edit-validator --from=<key name> --moniker=<your_custom_name> --website=<your_website> --details=<your_details> --chain-id=<chain-id> --node=tcp://localhost:26657 --fee=0.3iris  
 
-View the validator's information with this command:
-
-```
-iriscli stake validator <val-address-operato> --chain-id=<chain-id>
 ```
 
 ### Use IRISPlorer
```
Comment (haifengxi) [inline docs/get-started/Validator-Node.md]: Use the Explorer

File: docs/software/How-to-install-Irishub.md [hunk 16]
```diff
@@ -8,6 +8,19 @@ Please replace <latest_iris_version> with v0.13.1 while using "git checkout"
 
 You can download the source code from github and compile it locally.
 
+#### Configure Your Server
+
+All the blockchains in IRISnet are based on Cosmos-SDK, which is a framework for building blockchain applications in Golang. It is being used to build [Cosmos Hub](https://cosmos.network/). It's recommended to run a validator node on Linux server.
+
```
Comment (haifengxi) [inline docs/software/How-to-install-Irishub.md]: Start this paragraph with "It's recommended that you run a validator node on Linux Server." and remove the two sentences before it.

File: docs/.vuepress/config.js (hunk 3 above)
Comment (shilei-wang) [reply docs/.vuepress/config.js]: fixed

File: docs/get-started/Join-the-Testnet.md [hunk 17]
```diff
@@ -2,19 +2,19 @@
 
 ## Fuxi  Testnet
```
[the remark this answers is not in the document]
Comment (shilei-wang) [reply docs/get-started/Join-the-Testnet.md]: fixed

File: docs/get-started/README.md (hunk 4 above)
Comment (shilei-wang) [reply docs/get-started/README.md]: fixed

File: docs/get-started/README.md (hunk 6 above)
Comment (shilei-wang) [reply docs/get-started/README.md]: fix

File: docs/get-started/README.md (hunk 5 above)
Comment (shilei-wang) [reply docs/get-started/README.md]: fixed

File: docs/get-started/README.md (hunk 7 above)
Comment (shilei-wang) [reply docs/get-started/README.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 8 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 9 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 10 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 11 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 12 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 13 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 15 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/software/How-to-install-Irishub.md (hunk 16 above)
Comment (shilei-wang) [reply docs/software/How-to-install-Irishub.md]: fixed

File: docs/get-started/Validator-Node.md (hunk 14 above)
Comment (shilei-wang) [reply docs/get-started/Validator-Node.md]: fixed

File: docs/zh/get-started/Validator-Node.md (hunk 2 above)
Comment (shilei-wang) [reply docs/zh/get-started/Validator-Node.md]: fixed

**Pull request merged**